intel-retail/automated-self-checkout
# docs: Fix broken links, remove outdated references

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable
run `make serve-docs` and ensure there are no warnings about links that cannot be resolved

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
